kernelci/configs.py

```diff
@@ -523,6 +524,10 @@ def boot_method(self):
     def dtb(self):
         return self._dtb
 
+    @property
+    def params(self):
+        return self._params
```

Comment:
Again, see how it's done in `TestPlan.params()`:
```python
        return dict(self._params)
```
Returning a copy is to prevent changing the contents of the dictionary in the object.